tests/tests/test_cells/test_cells.py

```diff
@@ -101,3 +68,149 @@ def test_group_cells_by_z(xml_path):
         len(cells_groups[plane]) for plane in z_planes_test
     ]
     assert cell_numbers_in_groups_validate == cell_numbers_in_groups_test
```

Comment:
```suggestion
    assert  cell_numbers_in_groups_test == cell_numbers_in_groups_validate
```
More nitpicking (sorry again): Follow `assert actual == expected` convention for legibility?
(a bit like [Yoda conditions](https://blog.codinghorror.com/new-programming-jargon/))